cmd/state-svc/internal/messages/messages.go

```diff
@@ -140,11 +165,23 @@ func check(params *ConditionParams, messages []*graph.MessageInfo, lastReportMap
 			}
 		}
 
+		// Check if message is within date range
+		inRange, err := messageInDateRange(message, baseTime)
+		if err != nil {
+			logging.Warning("Could not check if message %s is in date range: %v", message.ID, err)
+			continue
+		}
+		if !inRange {
+			logging.Debug("Skipping message %s as it is outside of its date range", message.ID)
+			continue
+		}
+
 		// Validate the conditional
 		if message.Condition != "" {
 			result, err := strutils.ParseTemplate(fmt.Sprintf(`{{%s}}`, message.Condition), params, funcMap)
 			if err != nil {
-				return nil, errs.Wrap(err, "Could not parse condition template for message %s", message.ID)
+				logging.Warning("Could not parse condition template for message %s: %v", message.ID, err)
```

Comment:
Is there a risk this will cause us to miss potential issues that Rollbar would otherwise notify us of?

Reply:
Actually, never mind, because this is covered in the last AC. I'm good.

Reply:
Yeah, I waffled a bit on this too until I reread the AC.